.github/workflows/cibuild.yml

```diff
@@ -20,10 +20,10 @@ jobs:
         sudo apt-get update
         sudo apt-get install bwa samtools fastqc
         curl -L -O http://ccb.jhu.edu/software/FLASH/FLASH-1.2.11-Linux-x86_64.tar.gz && tar -xzf FLASH-1.2.11-Linux-x86_64.tar.gz && sudo cp FLASH-1.2.11-Linux-x86_64/flash /usr/local/bin/ && rm -r FLASH-1.2.11*
-        curl -L https://osf.io/wuzm6/download --output bwa_index.tar.gz && tar -xzf bwa_index.tar.gz && cp bwa_index/* microhapulator/data && rm -r bwa_index*
         python -m pip install --upgrade pip
         pip install .
         make devdeps
+        curl -L https://osf.io/wuzm6/download --output bwa_index.tar.gz && tar -xzf bwa_index.tar.gz && cp bwa_index/* /home/runner/work/MicroHapulator/MicroHapulator/microhapulator/data/ && rm -r bwa_index*
```

Comment:
Moving this download to later in the build. Putting those files in place *before* calling `pip install` results in an attempt to include the files when building the package "wheel", which grinds to a halt and overwhelms the storage capacity of the CI VM.